packages/client/src/index.ts

```diff
@@ -6,18 +6,6 @@ export interface ConfigParameters {
   clientId: string;
 }
 
-export const extractBearerToken = (authorization: string): string => {
-  if (
-    !authorization ||
-    (!authorization.startsWith('Bearer ') && !authorization.startsWith('bearer '))
```

Comment:
nit: Just  FYI If I remember correctly `authorization` header namespace as well as `Bearer` keyword are case insensitive.  In other words, feels like `BEARER` might be valid as well.  Better use `toLowerCase()` here.